docs/Files/UI/Solution-Manager/Managing-Agents.md

```diff
@@ -0,0 +1,13 @@
+---
+lang: en-us
+title: Managing Agents
+viewport: width=device-width, initial-scale=1.0
+---
+
+# Managing Agents
+
+See [Performing-Agent-Status-Updates](Performing-Agent-Status-Updates.md).
+
+![Managing Agents](../../../Resources/Images/SM/Library/ManagingLibrary/ManagingGlobalProperties.png "Threshold Grid")
```

Comment:
we should update all image links to display the correct image. for example, this one will display the global properties image instead of an agents image.

Reply:
Fixed and adjusted as needed all over